components/function-controller/internal/controllers/kubernetes/secret_controller_test.go

```diff
@@ -28,9 +30,14 @@ var _ = ginkgo.Describe("Secret", func() {
 
 		baseSecret = newFixBaseSecret(config.BaseNamespace, "ah-tak-przeciez")
 		gomega.Expect(resourceClient.Create(context.TODO(), baseSecret)).To(gomega.Succeed())
-
 		request = ctrl.Request{NamespacedName: types.NamespacedName{Namespace: baseSecret.GetNamespace(), Name: baseSecret.GetName()}}
 		reconciler = NewSecret(k8sClient, log.Log, config, secretSvc)
+
+		baseSecretWithManagedLabel = newFixBaseSecretWithManagedLabel(config.BaseNamespace, "secret-with-managed-label")
+		gomega.Expect(resourceClient.Create(context.TODO(), baseSecretWithManagedLabel)).To(gomega.Succeed())
+		requestForSecretWithManagedLabel = ctrl.Request{NamespacedName: types.NamespacedName{Namespace: baseSecretWithManagedLabel.GetNamespace(), Name: baseSecretWithManagedLabel.GetName()}}
+		reconciler = NewSecret(k8sClient, log.Log, config, secretSvc)
```

Comment:
you're overwriting this reconciler again, it's created in line 33

Reply:
Done